tensorflow_addons/layers/tests/normalizations_test.py

```diff
@@ -460,3 +460,13 @@ def test_eps_gards(dtype):
     model.fit(random_inputs, random_labels, epochs=1)
     final_eps_value = frn.eps_learned.numpy()[0]
     assert initial_eps_value != final_eps_value
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+@pytest.mark.parametrize("dtype", [np.float16, np.float32, np.float64])
+def test_filter_response_normalization_save_h5(dtype, tmpdir):
+    input_layer = tf.keras.layers.Input(shape=(32, 32, 3))
+    frn = FilterResponseNormalization()(input_layer)
+    model = tf.keras.Model(input_layer, frn)
+    filepath = str(tmpdir / "test.h5")
```

Comment:
Is this specific to h5?

Reply:
Not really. Thanks!

Reply:
Just as a side note I don't know if `tf.keras.models.model_to_json()` could be enought.